packages/fields-auto-increment/package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "@babel/runtime": "^7.8.4",
     "@keystonejs/adapter-knex": "^11.0.0",
+    "@keystonejs/adapter-prisma": "^0.0.0",
```

Comment:
This is what we should move away from. The way each adapter related code is required in each field (almost)

I wish we could have some spec with which the base adapter would expose the transformation functions and we use that in field and new adapter can provide such thing directly. 

if you look at base adapter in Keystone, that has bunch of high level api and each implementing adapter exposes them as private/internal functions.

Reply:
I 100% agree with you here, but let's track / discuss that separately to this PR 👍